src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/page.tsx

```diff
@@ -37,22 +54,58 @@ export default function Module() {
   }
 
   return (
-    <div>
-      <Link href={`/managed-courses/${courseId}/modules/${moduleId}/edit`}>Edit</Link> /
-      <button
-        type="button"
-        onClick={() => deleteModule(courseModule.id)}
-        disabled={deletingModule}
-      >
-        Delete
-      </button>
-      <section>
-        <p>ID: {courseModule.id}</p>
-        <p>Title: {courseModule.title}</p>
-        <p>Description: {courseModule.description}</p>
-        <p>Type: {courseModule.type}</p>
-        <p>Content: {courseModule.content}</p>
-      </section>
-    </div>
+    <CRow>
+      <CCol>
+        <CCard className="mb-4">
+          <CCardBody>
+            <CRow>
+              <CCol>
+                <CCardTitle className="fw-semibold fs-4">{courseModule.title}</CCardTitle>
+              </CCol>
+              <CCol xs="auto">
+                <CButton
+                  color="light"
+                  className="me-2"
+                  href={`/managed-courses/${courseId}/modules/${moduleId}/edit`}
+                >
+                  <CIcon icon={cilPencil} size="sm" className="" /> Edit
+                </CButton>
+                <CButton color="danger" onClick={() => setVisible(!visible)} className="text-light">
```

Comment:
This `setVisible(!visible)` is not recommended.
Use `state(state => !state)`


Read: https://medium.com/@shiny.shubham/how-to-update-state-in-react-the-correct-way-4ecb2f24f9b8
![Screenshot 2024-11-14 at 1 00 33 PM](https://github.com/user-attachments/assets/d6c42aad-d4d2-4815-bb7f-3feb2855a4b5)